lib/awspec/helper/finder.rb

```diff
@@ -148,6 +148,16 @@ module Finder
       http_proxy: ENV['http_proxy'] || ENV['https_proxy'] || nil
     }
 
+    # define_method below will "hide" any exception that comes from bad
+    # setup of AWS client, so let's try first to create a instance
+    if File.exist?(ENV['HOME'] + '/.aws') && File.directory?(ENV['HOME'] + '/.aws')
```

Comment:
awspec should not check `.aws`. I think this is beyond the awspec's responsibilities

Reply:
Just removed the `.aws` verification... now awspec will try to create a instance of `Aws::EC2::Client` in any setup situation (`.aws` or environment variables).